create3_lidar/config/mapper_params_online_async.yaml

```diff
@@ -1,4 +1,5 @@
-slam_toolbox:
+# slam_toolbox
+/**:
```

Comment:
keep the name of the node, otherwise there may be conflicts between parameters.
see here as an example https://github.com/iRobotEducation/create3_sim/blob/main/irobot_create_common/irobot_create_common_bringup/config/hazard_vector_params.yaml#L2-L3